app/controllers/admin/api/application_plans_controller.rb

```diff
@@ -87,6 +87,10 @@ def show
   ##~ op.parameters.add @parameter_application_plan_id_by_id
   ##~ op.parameters.add :name => "name", :description => "Name of the application plan.", :dataType => "string", :paramType => "query"
   ##~ op.parameters.add :name => "approval_required", :description => "Set the 'Applications require approval?' to 'true' or 'false'", :dataType => "boolean", :required => false, :paramType => "query"
+  ##~ op.parameters.add :name => "cost_per_unit", :description => "Cost per unit", :dataType => "decimal", :required => false, :paramType => "query"
```

Comment:
I think `cost_per_unit` is an attribute of `PricingRule`, not `ApplicationPlan`.

Reply:
Maybe you meant `cost_per_month`.

Reply:
right. fixed.